docs/content/concepts/automation/schedules/testing.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: "Testing schedules | Dagster Docs"
+description: "Learn to test your schedules in Python or the Dagster UI."
+---
+
+# Testing schedules
+
+In this guide, we'll show you how to use the Dagster UI and Python to test your [schedules](/concepts/partitions-schedules-sensors/schedules). For more information about testing in Dagster, refer to the [Testing documentation](/concepts/testing).
+
+---
+
+## Testing schedules in the Dagster UI
+
+Using the UI, you can manually trigger test evaluations of a schedule and view the results. This can be helpful when [creating a schedule](/concepts/automation/schedules/automating-assets-schedules-jobs) or for [troubleshooting unexpected scheduling behavior](/concepts/automation/schedules/troubleshooting).
```

Comment:
Thoughts on flipping these sentences-ish? To convey the problem first and then what to do to solve it:

```suggestion
When [creating a schedule](/concepts/automation/schedules/automating-assets-schedules-jobs) or for [troubleshooting unexpected scheduling behavior](/concepts/automation/schedules/troubleshooting), it can be helpful to verify the logic of how your schedule evaluates when to request a run. Using the UI, you can manually trigger test evaluations of a schedule and view the results. 
```